### PR TITLE
[CW-423] Added version to format property validation

### PR DESCRIPTION
**Pull request by michalzurawski**

### CODE REVIEW

File: schema/formats.go [hunk 1]
```diff
@@ -95,4 +102,5 @@ func registerGohanFormats(checkers gojsonschema.FormatCheckerChain) {
 	checkers.Add("port", portFormatChecker{})
 	checkers.Add("yaml", yamlFormatChecker{})
 	checkers.Add("text", textFormatChecker{})
+	checkers.Add("version", versionFormatChecker{})
```
Comment (p-kozlowski) [inline schema/formats.go]: please add it to the docs

Comment (michalzurawski) [reply schema/formats.go]: Done

**Comment by michalzurawski** [comment]
Build fails due to missing semver in vendor. I'll wait for https://github.com/cloudwan/gohan/pull/658 with adding it

**Comment by morrisson** [comment]
#658 has been merged but still semver needs to be included.

**Comment by morrisson** [comment]
@michalzurawski As you may know, `github.com/Masterminds/semver` should be stored in `vendor` directory.

**Comment by michalzurawski** [comment]
will wait with this PR as there is still pending discussion - as you can see in the tests semver allows using i.e. "1", "1.1" or "v1.1.1" which probably doesn't suit our case

**Pull request merged**